src/core/doc_generators/models.py

```
@@ -592,22 +575,22 @@ def process_batch(self, config: BatchGenerationConfig) -> BatchGenerationResult:
     def process_directory(
         self,
         input_directory: str | Path,
-        output_directory: str | Path,
+        output_path: str | Path,
         config: DocumentConfig | None = None,
```

Ruff (0.14.0)

Unused method argument: `config`

(ARG002)